openedx/core/djangoapps/user_authn/templates/user_authn/edx_ace/passwordreset/email/body.html

```diff
@@ -3,7 +3,7 @@
 {% load i18n %}
 {% load static %}
 {% block content %}
-<table width="100%" align="left" border="0" cellpadding="0" cellspacing="0" role="presentation">
+<table width="100%" align="left" border="0" cellpadding="0" cellspacing="0" role="presentation" style="box-shadow: 0 1px 5px rgba(0,0,0,0.25);">
```

Comment:
This not the correct place for this, kindly check the screenshot.
<img width="1259" alt="Screenshot 2021-03-02 at 3 04 08 PM" src="https://user-images.githubusercontent.com/2851134/109632234-9dd92100-7b68-11eb-8e6f-82667a257bf0.png">

Actually, the box-shadow is already set in the base template `ace_common/edx_ace/common/base_body.html`, you might need to investigate why it's working in other emails but not in this one.